src/daft-csv/src/python.rs

```diff
@@ -31,6 +31,9 @@ pub mod pylib {
         has_header: Option<bool>,
         delimiter: Option<&str>,
         double_quote: Option<bool>,
+        quote: Option<char>,
+        escape_char: Option<char>,
+        comment: Option<char>,
```

Comment:
Great use of char here! I wasn't aware that pyo3 had `PyString` -> `char` conversions, I'll have to update `delimiter` to use that as well.

Reply:
I'll incorporate this change as well.

Reply:
I had my doubts about this, as well. I [asked about this](https://github.com/PyO3/pyo3/discussions/3569) in pyo3, seems like this was just not documented.